packages/tpetra/core/test/CrsMatrix/CMakeLists.txt

```diff
@@ -47,6 +47,15 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
   STANDARD_PASS_OUTPUT
   )
 
+TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  CrsMatrix_UnitTests5
```

Comment:
Is there anything special about why we group these particular tests together, that would let us give a more descriptive name than "5"?

Reply:
Renamed CrsMatrix_UnitTests_LocalAccessors